var/spack/repos/builtin/packages/wgrib2/package.py

```diff
@@ -37,27 +82,48 @@ class Wgrib2(MakefilePackage):
         extension="tar.gz",
     )
 
-    depends_on("c", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
```

Comment:
We should keep the language lines for the compilers-as-dependencies functionality.

Reply:
Agree, thanks for catching that. Fixed.